src/option_parser.cr

```diff
@@ -278,8 +278,10 @@ class OptionParser
   end
 
   private def append_flag(flag, description)
+    indent = " " * 37
```

Comment:
I'm pretty sure that fixed number won't work well with flags that are less than 33 characters long.

Reply:
This indent is only used for the description or in case flags are longer than 33 characters.
In the test spec both cases are tested (flags.size < 33 and flags.size > 33).

Reply:
Sorry, I was mistaken about how the indent works for flag.size < 33.

Reply:
Isn't it used in the following line (before the `if flag.size >= 33`)?